norman_objects/shared/models/asset_name.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class AssetName(str, Enum):
```

Comment:
Question: Isn't it about time we open a `/shared/assets` directory for everything asset-related?

If so, move everything there.